From 240f6e6698cf72ea40925b12b59e57f867ff2dec Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Mon, 15 Sep 2014 10:52:32 +0200
Subject: [PATCH 13/38] Increase default key length for newly-generated RSA
 keys

Should we add something for chrome://settings, or just leave
it as hidden as it was before?
---
 net/cert/x509_util.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/cert/x509_util.cc b/net/cert/x509_util.cc
index d74340e..7dd92b3 100644
--- a/net/cert/x509_util.cc
+++ b/net/cert/x509_util.cc
@@ -16,7 +16,7 @@ namespace net {
 namespace x509_util {
 
 // RSA keys created by CreateKeyAndSelfSignedCert will be of this length.
-static const uint16 kRSAKeyLength = 1024;
+static const uint16 kRSAKeyLength = 2048;
 
 // Certificates made by CreateKeyAndSelfSignedCert and
 //  CreateKeyAndChannelIDEC will be signed using this digest algorithm.
-- 
2.4.0

